app/client/src/couchers/__init__.py

```diff
@@ -0,0 +1 @@
+from .client import get_client  # noqa
```

Comment:
post-merge comment: Having an `__init__.py` in `couchers` prevents us from using `couchers.foo` in the backend since it is no longer a PEP420 namespace. I'd rather avoid painitng ourselves into that corner and remove this file.

I can't test when I'm on vacation but a simple `pip install couchers` while in the backend venv will show the problems I'm talking about.